src/picknik_ur_base_config/objectives/request_teleoperation.xml

```diff
@@ -1,42 +1,72 @@
 <?xml version='1.0' encoding='UTF-8'?>
 <root BTCPP_format="4" main_tree_to_execute="Request Teleoperation">
-  <!-- The integer value used here for teleoperation mode comes from the moveit_studio_sdk_msgs/TeleoperationMode ROS message. -->
+  <!--The integer value used here for teleoperation mode comes from the moveit_studio_sdk_msgs/TeleoperationMode ROS message.-->
   <BehaviorTree ID="Request Teleoperation" _description="Handles the different variations of teleoperation from the web UI, with the option of interactive user prompts and choosing the initial mode. Should be used as a subtree with port remapping as part of an another Objective." _favorite="false" _hardcoded="false">
     <Control ID="Sequence">
       <Action ID="Script" code="teleop_mode := 0"/>
       <Control ID="Parallel" success_count="1" failure_count="1">
         <Action ID="DoTeleoperateAction" enable_user_interaction="{enable_user_interaction}" user_interaction_prompt="{user_interaction_prompt}" initial_teleop_mode="{initial_teleop_mode}" current_teleop_mode="{teleop_mode}"/>
         <Decorator ID="KeepRunningUntilFailure">
           <Control ID="Sequence">
-            <!-- Closing and opening the gripper -->
+            <!--Closing and opening the gripper-->
             <Decorator ID="ForceSuccess" _skipIf="teleop_mode != 7">
               <SubTree ID="Close Gripper"/>
             </Decorator>
             <Decorator ID="ForceSuccess" _skipIf="teleop_mode != 6">
               <SubTree ID="Open Gripper"/>
             </Decorator>
-            <!-- Joint sliders interpolate to joint state -->
+            <!--Joint sliders interpolate to joint state-->
             <Decorator ID="ForceSuccess" _while="teleop_mode == 5">
               <Control ID="Sequence">
                 <Action ID="SwitchUIPrimaryView" primary_view_name="Visualization"/>
-                <SubTree ID="Interpolate to Joint State"/>
+                <Control ID="Fallback" name="root">
+                  <Control ID="Sequence">
+                    <Action ID="RetrieveJointStateParameter" timeout_sec="-1" joint_state="{target_joint_state}"/>
```

Comment:
Shouldn't we have some timeout to avoid getting stuck forever because of some bug?

Reply:
Is this a known bug or just in case? I think if the objective hangs you can always hit cancel.

Reply:
Yes, you could call cancel but wouldn't it be nicer to just fail after maybe 5s?

Reply:
Shouldn't block the PR though. Let's merge it :+1: